src/components/new-suggestion/suggestion-page-section.tsx

```diff
@@ -0,0 +1,151 @@
+import React, { useEffect, useState } from "react"
+import { XMarkIcon } from "@heroicons/react/24/outline"
+import ProgressBar from "@/components/new-suggestion/progress-bar/progress-bar"
+import SongInformationArea from "@/components/new-suggestion/areas/song-information.area"
+import ReviewArea from "@/components/new-suggestion/areas/review.area"
+import { Area } from "@/constants/area"
+import { FormProvider, useForm } from "react-hook-form"
+import {
+  InputsSongInformation,
+  NewSuggestion,
+  NewSuggestionInstrument,
+} from "@/interfaces/new-suggestion"
+import InstrumentsArea from "@/components/new-suggestion/areas/instruments/instruments.area"
+import { Database } from "@/types/database"
+import { Instrument } from "@/types/database-types"
+import { useSupabaseClient } from "@supabase/auth-helpers-react"
+import { getInstruments } from "@/services/instrument.service"
+import Spinner from "@/components/utils/spinner"
+import ErrorPopup from "@/components/popups/error-popup"
+import { submitSongInformationForm } from "@/helpers/new-suggestion.helper"
+
+interface SuggestionPageSectionProps {
+  title: string
+  newSuggestion: NewSuggestion
+  startingArea: Area
+  onCloseClicked(): void
+  onSongInformationSubmit(songInformation: InputsSongInformation): void
+  onInstrumentSubmit(newInstruments: NewSuggestionInstrument[]): void
+  onAreaSelect(newArea: Area): void
+  onReviewSubmit(onSuccess: () => void, onError: () => void): void
```

Comment:
Do the onSucccess & onError have any functionality

Reply:
![image](https://github.com/ASVGay/the-rhapsodies/assets/4838158/26de20ed-b021-41c4-b647-6638517d7056)
Used in reviewArea